src/util/value.ts

```diff
@@ -80,6 +80,20 @@ const valueTypeMapper: Partial<Record<GQLValueType, ValueType>> = {
 
 export type ValueType = keyof ValueTypes;
 
+const enumTypeMapper: Partial<Record<any, string>> = {
+  [Theme.Default]: "default",
+  [Theme.Dark]: "dark",
+  [Theme.Light]: "light",
+  [AssetSortType.Date]: "date",
+  [AssetSortType.Name]: "name",
+  [AssetSortType.Size]: "size",
+};
```

Comment:
1. Split (theme and asset type are entirely separate concerns)

```suggestion
const enumTheme = {
  [Theme.Default]: "default",
  [Theme.Dark]: "dark",
  [Theme.Light]: "light",
}

const enumAsserType = {
  [AssetSortType.Date]: "date",
  [AssetSortType.Name]: "name",
  [AssetSortType.Size]: "size",
};
```

2. Transfer them to each organisms hooks, because `src/util/value.ts` is the place to implement conversion between only "Property Values" ("value" in the file name means "property values").

Reply:
Gotcha, makes sense. Thanks